client/landing/stepper/declarative-flow/internals/steps-repository/playground/components/playground-iframe/index.tsx

```diff
@@ -13,6 +15,7 @@ export function PlaygroundIframe( {
 } ) {
 	const iframeRef = useRef< HTMLIFrameElement >( null );
 	const recommendedPHPVersion = usePhpVersions().recommendedValue;
+	const [ , setSearchParams ] = useSearchParams();
```

Comment:
```suggestion
	const [ setSearchParams ] = useSearchParams();
```

Reply:
Don't think this is correct. We need to skip the first returned value and assign second returned value to the const. Can you explain?

Reply:
> Don't think this is correct. We need to skip the first returned value and assign second returned value to the const. Can you explain?

Yes, it's the current version. We only need the set function.

Reply:
> @zaerl Do you mean to say the current version is correct or the suggestion version is?

The current one is right.